utils/pgUtils.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const pgp = require('pg-promise');
+const connectionConfig = {
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  database: process.env.DB_NAME,
+  user: process.env.DB_USER,
+  password: process.env.DB_PW,
```

Comment:
In fileQcDao.js this block turned into `const db = dbUtils.db;`, why not here?

Reply:
This file exports `db`, which is imported by `fileQcDao` as `dbUtils`. (`dbUtils` is also imported by `caseDao`)

Reply:
I also have another commit that cleans up the imports; I'll cherry-pick that onto this branch.